marlowe-runtime/src/Language/Marlowe/Runtime/Transaction/Api.hs

```diff
@@ -346,13 +350,21 @@ newtype CreateError
 data ApplyInputsError
   = ApplyInputsUnsolvableConstraints UnsolvableConstraintsError
   | ScriptOutputNotFound
+  | ApplyInputsLoadMarloweContextFailed LoadMarloweContextError
   deriving (Eq, Show, Generic, Binary)
 
-newtype WithdrawError
+data WithdrawError
   = WithdrawUnsolvableConstraints UnsolvableConstraintsError
```

Comment:
The name `WithdrawUnsolvableConstraints` implies that they are provably unsolvable rather than the actual case is that Runtime couldn't solve the constraints. Maybe this distinction doesn't matter.

Reply:
We can change it later if we want to